Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignUpScreen.kt

```diff
@@ -58,7 +61,10 @@ fun SignUp(onNavigationEvent: (SignUpEvent) -> Unit) {
         content = {
             SignInSignUpScreen(
                 onSignedInAsGuest = { onNavigationEvent(SignUpEvent.SignInAsGuest) },
-                modifier = Modifier.fillMaxWidth()
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .wrapContentWidth(align = Alignment.CenterHorizontally)
+                    .widthIn(max = 840.dp)
```

Comment:
Same modifier as above. Maybe it's worth extracting this modifier out to a global variable? It'll have a better semantic meaning and it can be self-documented